### ISSUE TITLE
Do I need to change the hyperparameters for training?

### ISSUE THREAD

**Issue by yipliu** [SilongYong/SQA3D#2]
I have run [auxi.sh](https://github.com/SilongYong/SQA3D/blob/master/ScanQA/auxi.sh) on 1 3090 GPU, and the following results were obtained

`best.txt`
--------------------------------------best--------------------------------------
[best] epoch: 31
[loss] loss: 13.506192174612307
[loss] vote_loss: 0.4446525583956756
[loss] objectness_loss: 0.2101170407498584
[loss] box_loss: 0.2650826440430155
[loss] sem_cls_loss: 1.041063628360337
[loss] aux_loss: 1.1851836922121983
[loss] answer_loss: 3.1320075708277084

[sco.] answer_acc@1: 0.45951168940347786
[sco.] answer_acc@10: 0.8645833333333334

`acc on test`
```python
"acc_at1": {
  "What_acc": 29.64,
  "Is_acc": 62.63,
  "How_acc": 45.81,
  "Can_acc": 67.75,
  "Which_acc": 39.03,
  "Others_acc": 32.53,
  "overall_acc_1": 44.59}
```
However, I obtained the same results as the paper using your publicly available model.

```python
  "acc_at1": {
    "What_acc": 33.74,
    "Is_acc": 66.79,
    "How_acc": 41.51,
    "Can_acc": 69.53,
    "Which_acc": 42.45,
    "Others_acc": 35.2,
    "overall_acc_1": 47.14
  }
```
Therefore, I doubt that the hyparameters must be modified during training in [auxi.sh](https://github.com/SilongYong/SQA3D/blob/master/ScanQA/auxi.sh)

**Comment by jeasinema** [comment]
Hey @yipliu,

I believe this is caused by the discrepany on model selection. After checking our pipeline, the model we released follows a different way of model selection (not based on the best model on validation). For now, you may continue to work on your own model as we're tunning the baseline to make sure it can match the released model but still using the model selected using the validation set. Will keep you posted and sorry for the confusion!

Best,
Xiaojian
